scipy/sparse/_coo.py

```diff
@@ -367,6 +367,19 @@ def tocsr(self, copy=False):
                 x.sum_duplicates()
             return x
 
+    def _coo_to_compressed(self, swap):  #major_dim, minor_dim, major, minor, data):
```

Comment:
Seems like we should be calling this inside the `tocsc()` and `tocsr()` methods above.

Reply:
Also, I assume we don't need the comment here.